src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -301,6 +313,28 @@ export default class MutationTable<P extends IMutationTableProps> extends React.
             visible: DiscreteCNAColumnFormatter.isVisible(this.props.discreteCNACache as DiscreteCNACache)
         };
 
+        this._columns[MutationTableColumnType.FACETS_COPY_NUM] = {
+            name: "Integer Copy #",
```

Comment:
You should consider moving all these strings to constants so they can be shared by two pages. Even better you write a template for these columns and only overwrite properties when necessary

Reply:
Well, we should have initially implemented the `MutationTable` with something like this: https://github.com/cBioPortal/react-mutation-mapper/blob/a08cc8381fe67ba8cefbf5f55cb8adf74caa477e/src/MutationColumnHelper.tsx#L28